pilot/pkg/bootstrap/server.go

```diff
@@ -513,16 +517,13 @@ func (c *mockController) Run(<-chan struct{}) {}
 
 func (s *Server) initMCPConfigController(args *PilotArgs) error {
 	clientNodeID := ""
-	collections := make([]sink.CollectionOptions, len(model.IstioConfigTypes))
-	for i, t := range model.IstioConfigTypes {
-		collections[i] = sink.CollectionOptions{Name: t.Collection, Incremental: false}
-	}
-
-	options := coredatamodel.Options{
-		DomainSuffix: args.Config.ControllerOptions.DomainSuffix,
-		ClearDiscoveryServerCache: func() {
-			s.EnvoyXdsServer.ConfigUpdate(true)
-		},
+	var collections []sink.CollectionOptions
+	for _, c := range model.IstioConfigTypes {
+		// do not register SSEs for this controller as there is a dedicated controller
```

Comment:
If you've got a separate controller for SSE's, why do we need to modify this controller?

Reply:
This is just to exclude or not register the SSE type from the regular controller.

Reply:
I ask because I see a lot of changes to this file ... maybe revert your changes?

Reply:
These changes in `bootstrap/server` are required. This is where we register both controllers (regular and SSE). Just to summarise the changes to this file; 
- excludes SyntheticServiceEntry type from the regular controller
- initialise necessary arguments for the new sse controller
- register the xdsUpdater for both controllers

Reply:
oh sorry ... forgot which file this was. :)